pkgs/top-level/aliases.nix

```diff
@@ -10,6 +10,11 @@ lib: self: super:
 
 # A script to convert old aliases to throws and remove old
 # throws can be found in './maintainers/scripts/remove-old-aliases.py'.
+# We aim to run it after each nixos release branchoff. Feel free to
```

Comment:
Nitpick: Who is 'we'? The release manager? If so, this should probably be documented in whatever checklist is used. If it's not a defined person at all, I'd be surprised if this ever happens.

Alternatively, there could be a CI that periodically runs the script and opens a new PR. If this is the route that ends up happening, I don't think it would be out of scope to mention that intention here.